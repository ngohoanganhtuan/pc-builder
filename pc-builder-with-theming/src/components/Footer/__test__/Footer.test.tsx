import * as renderer from 'react-test-renderer';

// Components
import { Footer } from '@components';

describe('Test [Footer] component', () => {
  test('Component [Footer] render correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
