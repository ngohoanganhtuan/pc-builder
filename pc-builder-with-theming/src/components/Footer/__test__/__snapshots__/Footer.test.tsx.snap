// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test [Footer] component Component [Footer] render correctly 1`] = `
<footer
  className="shadow-sm border bg-white footer"
>
  <div
    className="container p-3 bg-indigo-500"
  >
    <h2
      className="fs-4 title"
    >
      PC Builder
    </h2>
    <p
      className="description"
    >
      PC Builder and the PC Builder logo are registered patents and trademarks of PC Builder. All other names, brands, products or services are trademarks or registered trademarks of their respective owners.
    </p>
    <div
      className="icon-list list-group"
    >
      <div
        className="d-flex flex-row gap-2"
      >
        <i
          className="bi bi-envelope icon"
        />
        <a
          className="text-decoration-none text-dark link"
          href="mailto:none"
        >
          pcbuilder@gmail.com
        </a>
      </div>
      <div
        className="d-flex flex-row gap-2"
      >
        <i
          className="bi bi-telephone icon"
        />
        <a
          className="text-decoration-none text-dark link"
          href="tel:none"
        >
          0236-0000-000
        </a>
      </div>
    </div>
  </div>
</footer>
`;
